### ISSUE TITLE
How to remove "date updated" on posts

### ISSUE THREAD

**Issue by Stefany93** [understrap/understrap#810]
## Issue is about..
  - [ ] I need help

# Issue Overview for Bugreports

## Screenshots / Video
![image](https://user-images.githubusercontent.com/2403939/45631417-75284400-ba60-11e8-8dd3-1cf97b9af925.png)

## Expected behavior
Only the published one date must be shown

### Other Comments
Apperantly the `understrap_posted_on()` function controls the date posted / updated notices, but is there a way to make it only show the published date instead of published and updated date like it is now? A way that doesn't involve CSS. Thanks so much

**Comment by holger1411** [comment]
Hi!
You can find the function itself in inc/template-tags.php.
It starts from line 17:
https://github.com/understrap/understrap/blob/master/inc/template-tags.php#L17
The end of Line 21 includes the markup for the "updatwd" time:
<time class="updated" datetime="%3$s"> (%4$s) </time>

Trim this or remove it.

If you use a child theme (recommended): Copy the whole function from L18-38 to the child themes function.php. And than do the changes.

**Comment by IanDelMar** [comment]
@Stefany93 I had the same issue. Once (if) my PR #813 is merged into the master, you can use a filter to achieve that.

**Comment by Stefany93** [comment]
@holger1411 Thanks so much yes I have a child theme will try that. 

@IanDelMar  Thanks for making a PR and adding the filter option, hopefully it gets approved soon!

**Comment by holger1411** [comment]
Thx @Stefany93 and @IanDelMar (for bringing up this issue and fixing it) !
Closing this
